tests-real/suites/options/autocomplete/autocomplete_help.erl

```diff
@@ -0,0 +1,10 @@
+-module(autocomplete_help).
+
+-export([test/0]).
+
+test() ->
+  OptionSpec = concuerror_options:options(),
+  OptsRaw = [Keywords || {_, Keywords, _, _, _, _} <- OptionSpec],
+  Opts = lists:usort(lists:append([[all], [progress], [attributes] | OptsRaw])),
```

Comment:
Not that it makes any big difference, but perhaps create a single list with these three options instead of three here?

Reply:
Good point! Fixed.